contentcuration/contentcuration/frontend/channelEdit/views/files/supplementaryLists/SupplementaryList.vue

```diff
@@ -149,7 +149,7 @@
 
   .languageTile:hover {
     /* stylelint-disable-next-line custom-property-pattern */
-    background-color: var(--v-greyBackground-base);
```

Comment:
I believe this CSS var came from configuration in the Vuetify theme. We should slowly move away from this pattern (meaning removing it is a good direction), and instead use the Kolibri Design System (KDS) colors.